bids/modeling/transformations/munge.py

```diff
@@ -156,7 +156,7 @@ def _transform(self, var, constraint='none', ref_level=None, sep='.'):
                 continue
             name = ''.join([var.name, sep, str(lev)])
             lev_data = data.copy()
-            lev_data['amplitude'] = new_cols[lev].astype(float)
```

Comment:
Why did we have this in the first place but is no longer necessary?

Reply:
`get_dummies()` creates bools by default. This brought it back to float when reinserting into the dataframe. Instead, I make them floats at the beginning, since there's nothing that depends on them being bools above.